content/platform/zerto/home.md

```diff
@@ -1,34 +1,46 @@
 ---
 title: Zerto
-version: "1.0"
-description: "Data Protection as Code: Converged Backup, DR (Disaster Recovery),
-  and Mobility for Cloud Native Apps"
+version: "2.0"
+description: "Empowering an  run an always-on business by simplifying data
+  protection, recovery, and multi-cloud mobility "
 image: https://www.zerto.com/wp-content/uploads/2019/04/Zerto-Logo.png
 width: large
 priority: 11
 active: true
 ---
-Zerto, a Hewlett Packard Enterprise company, empowers customers to run an always-on business by simplifying the protection, recovery, and mobility of on-premises and cloud applications. Zerto’s cloud data management and protection solution eliminate the risks and complexity of modernization and cloud adoption across private, public, and hybrid deployments. The simple, software-only solution uses continuous data protection at scale to converge disaster recovery, backup, and data mobility. Zerto is trusted by over 9,500 customers globally and is powering offerings for Microsoft Azure, IBM Cloud, AWS (Amazon Web Services), Google Cloud, Oracle Cloud, and more than 350 managed service providers.  
+Zerto, a Hewlett Packard Enterprise company, empowers customers to run an always-on business by simplifying the protection, recovery, and mobility of on-premises and cloud applications. Zerto’s continuous data protection (CDP) eliminates the risks and complexity of modernization and cloud adoption across private, public, and hybrid deployments. The simple, software-only solution uses continuous data protection at scale to converge disaster recovery, backup, and data mobility. Zerto is trusted by over 9,500 customers globally and is powering offerings for Microsoft Azure, IBM Cloud, AWS (Amazon Web Services), Google Cloud, Oracle Cloud, and more than 350 managed service providers.  
 
 Learn more about [Zerto](https://www.hpe.com/us/en/zerto.html)
 
 ## Welcome to the Zerto Hacker Hub!
 
-Here you will be provided the tools to get started in deploying and managing Zerto’s continuous data protection solution that leverages its proprietary near-synchronous replication capabilities to protect and restore virtual machines, Kubernetes clusters, and public cloud instances. Zerto offers an extensive REST API with Swagger support, for users to programmatically create and manage end-to-end disaster recovery workflows.  
```

Comment:
Recommend rephrasing this slightly to:
**On this page,** you will be provided **with tools** **that will help you get** started in deploying and managing Zerto's CD**P, which** leverages...

For the second sentence, I'd modify part of it as well:
"Zerto offers an extensive REST API with Swagger suppor**t that enables** users to..."